addons/purchase/test/fifo_price.yml

```diff
@@ -343,7 +343,7 @@
     self.env['stock.immediate.transfer'].create({'pick_id': picking.id}).process()
     original_out_move = self.env['stock.picking'].browse(ref('outgoing_fifo_shipment_neg')).move_lines[0]
     assert original_out_move.remaining_qty == 50.0, 'On the out move of 100, 50 still needs to be matched'
-    assert original_out_move.value == 2500, 'Value of the move should be 2500'
+    assert original_out_move.value == -2500, 'Value of the move should be 2500'
```

Comment:
note: this change is because the negative stock was is wrongly implemented in saas-17 at the moment (in all cases, an out move should be negative)